src/app/sale/sale.class.ts

```diff
@@ -0,0 +1,19 @@
+
+
+
+export interface CakeSold{
```

Comment:
As a good practice, you can add the letter I before the name of your interfaces, to identify them versus the classes. Ex. `IBread` or `iBread`.